src/functionalTest/resources/features/F-004 PUT Hearing/S-004.1-get-hearing.td.json

```diff
@@ -0,0 +1,56 @@
+{
+  "_guid_": "S-004.1-get-hearing",
+  "_extends_" : "GetHearingRequestPositiveGeneric",
+  "specs": [
+    "to verify the party name has been updated"
+  ],
+  "request": {
+    "pathVariables": {
+      "hearingID" : "${[scenarioContext][siblingContexts][CreateHearingRequest][testData][actualResponse][body][hearingRequestID]}"
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 200,
+    "responseMessage": "OK",
+    "body": {
+      "requestDetails": {
```

Comment:
extract response data into separate file and extend that file here if the data is common enough

Reply:
Done in [2188106](https://github.com/hmcts/hmc-cft-hearing-service/pull/219/commits/218810605b79e9828fd4211d2f41169c7ce26897)